app/experimenter/nimbus-ui/src/components/ExperimentLoading/index.tsx

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import AppLayout from "../AppLayout";
+
+const ExperimentLoading = () => (
+  <AppLayout testid="experiment-loading">
```

Comment:
I'm down to call this something like `PageLoading` too if we'll reuse it elsewhere. I made this a full-page loading spinner inside of using it within `<AppLayoutWithSidebar>` since the sidebar links will change later based on experiment state... this is easiest for now.